docs/zero-to-hero/01-basics/01-set-up-skeleton.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 2
+sidebar_label: "Set up Rust and a contract skeleton" 
+title: "Set up Rust, get a NEAR testnet account, NEAR CLI, and get a basic smart contract skeleton ready"
+---
+
+# Getting started
```

Comment:
I know this may be a weird comment, but we should probably have a consistency about the capitalization of titles, looking around and seeing the differences across pages

Reply:
I see what you're saying. I believe I'm using the AP Style Guide:

>In AP Style, only the first word and proper nouns are capitalized in headlines.

https://writingexplained.org/ap-style/ap-style-headlines#:~:text=In%20AP%20Style%2C%20only%20the,no%20periods)%20in%20all%20headlines.

Perhaps we do need to sweep the rest of the docs later